src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.TargetingPackResolution.targets

```diff
@@ -0,0 +1,150 @@
+                        IsImplicitlyDefined="true"
+                        PrivateAssets="all"
+                        ExcludeAssets="all" />
+    </ItemGroup>
+
+  </Target>
+
+  <UsingTask TaskName="ReportUnknownFrameworkReferences" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+
+  <!-- Report unresolved FrameworkReferences in a separate target so that the error is only reported on Build, not Restore. -->
+  <Target Name="ReportUnresolvedFrameworkReferences" DependsOnTargets="ResolveFrameworkReferences"
+          BeforeTargets="_CheckForInvalidConfigurationAndPlatform"
+          Condition="'@(_UnresolvedFrameworkReference)' != ''">
+
+    <!-- Use a custom task in order to report multiple errors instead of stopping on the first one (which would happen if we batched) -->
+    <ReportUnknownFrameworkReferences UnresolvedFrameworkReferences="@(_UnresolvedFrameworkReference)"/>
+
+  </Target>
+
+
+  <UsingTask TaskName="GetPackageDirectory" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+  
+  <Target Name="ResolveTargetingPacks" DependsOnTargets="ResolvePackageAssets">
+
+    <GetPackageDirectory
+      Items="@(TargetingPack)"
+      ProjectPath="$(MSBuildProjectFullPath)"
+      PackageFolders="@(AssetFilePackageFolder)">
+
+      <Output TaskParameter="Output" ItemName="ResolvedTargetingPack" />
+      
+    </GetPackageDirectory>    
+    
+    <ItemGroup>
+      <ResolvedTargetingPack>
+        <Path Condition="'%(ResolvedTargetingPack.Path)' == ''">%(PackageDirectory)</Path>
+      </ResolvedTargetingPack>
+      <Reference Include="%(ResolvedTargetingPack.Path)\ref\netcoreapp3.0\*.dll">
+        <!-- TODO: Work out what metadata we should use here to display these references grouped under the targeting pack
+             in solution explorer.  These metadata values are based on what PCLs were using. -->
+        <WinMDFile>false</WinMDFile>
+        <Private>false</Private>
+        <ReferenceGroupingDisplayName>.NET Core</ReferenceGroupingDisplayName>
+        <ReferenceGrouping>%(ResolvedTargetingPack.Identity)</ReferenceGrouping>
+        <ResolvedFrom>ResolveTargetingPacks</ResolvedFrom>
+        <IsSystemReference>True</IsSystemReference>
+      </Reference>
+    </ItemGroup>
+      
+    <ItemGroup Condition="'$(RuntimeIdentifier)' == '' or '$(SelfContained)' != 'true'">
+      <PackageConflictPlatformManifests Include="%(ResolvedTargetingPack.Path)\build\netcoreapp3.0\Microsoft.NETCore.App.PlatformManifest.txt"
+                                        Condition="Exists('%(ResolvedTargetingPack.Path)\build\netcoreapp3.0\Microsoft.NETCore.App.PlatformManifest.txt')"/>
+    </ItemGroup>
+
+    <GetPackageDirectory
+      Items="@(AppHostPack)"
+      ProjectPath="$(MSBuildProjectFullPath)"
+      PackageFolders="@(AssetFilePackageFolder)">
+
+      <Output TaskParameter="Output" ItemName="ResolvedAppHostPack" />
+
+    </GetPackageDirectory>
+
+    <ItemGroup>
+      <ResolvedAppHostPack>
```

Comment:
The item is an output from the GetPackageDirectory task.  There's only supposed to be one, but I'm using an item so that I can set metadata on it instead of using multiple properties.

I've changed the condition on setting the property to this:

```xml
    <ItemGroup>
      <ResolvedAppHostPack>
        <Path Condition="'%(ResolvedAppHostPack.Path)' == ''">%(PackageDirectory)\%(RelativePath)</Path>
      </ResolvedAppHostPack>      
    </ItemGroup>

    <PropertyGroup Condition="'@(ResolvedAppHostPack)' != '' And '$(AppHostSourcePath)' == ''">
      <AppHostSourcePath>@(ResolvedAppHostPack->'%(Path)')</AppHostSourcePath>
    </PropertyGroup>
```

Does this match what you were getting at?

Reply:
You're right, I wasn't paying attention and didn't see that there's no `Include` on this item.  This is fine, so I'm resolving this comment.